python/cugraph/cugraph/tests/utils/test_dataset.py

```diff
@@ -165,7 +165,7 @@ def test_reader_dask(dask_client, dataset):
 
 @pytest.mark.parametrize("dataset", ALL_DATASETS)
 def test_get_edgelist(dataset):
-    E = dataset.get_edgelist()
+    E = dataset.get_edgelist(download=True)
```

Comment:
We can do this for now if it allows CI to pass, but we should look into why this is needed later.  [Here](https://github.com/rapidsai/cugraph/issues/5157)'s the issue for it.